torch/lib/TensorLib/THStorageView.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "TH/TH.h"
+
+namespace tlib {
+
+// make a fake storage out of a size, pointer pair...
+class THStorageView {
```

Comment:
Let's rename to THLongStorageView. I didn't bother making it generic because it is only used for size and stride passing.